extensions/common/wasm/json_util.h

```diff
@@ -38,6 +39,7 @@ template <typename T>
 std::pair<std::optional<T>, JsonParserResultDetail> JsonValueAs(
     const JsonObject&) {
   static_assert(true, "Unsupported Type");
+  return std::make_pair(std::nullopt, JsonParserResultDetail::UNKNOWN);
```

Comment:
Where does it come from? 
Does `static_assert(true,...) make sense? It seems it should be `static_assert(`*false* in the past and static_assert should be removed now...

Reply:
I thought I got a build error because of no return, but looks like that was not the case. Updated.